### PR TITLE
NCSD-2618: Adds script for setting values in shared configuration

### PR DESCRIPTION
**Pull request by thelious1974**

### CODE REVIEW

File: PSCoreScripts/Copy-SitefinityDatabase.ps1 [hunk 1]
```diff
@@ -42,7 +42,7 @@ if (!$SqlServerResource) {
 
 # extract the build number if it is not provided
 if (!$PSBoundParameters.ContainsKey("ReleaseNumber")) {
-    if ($ReleaseNumber = $ENV:RELEASE_RELEASENAME) {
+    if ($ReleaseNumber -eq $ENV:RELEASE_RELEASENAME) {
```
Comment (thelious1974) [inline PSCoreScripts/Copy-SitefinityDatabase.ps1]: this fixes a PSScriptAnalyzer error

File: PSCoreScripts/Copy-SitefinityDatabase.ps1 [hunk 2]
```diff
@@ -92,13 +92,12 @@ if (!$databaseCopyAlreadyExists) {
         CopyDatabaseName  = $CopyDatabaseName
     }
     if ($currentDatabase.SkuName -eq 'ElasticPool') {
-        $DatabaseCopyParameters['ElasticPoolName'] = $currentDatabase.ElasticPoolName 
+        $DatabaseCopyParameters['ElasticPoolName'] = $currentDatabase.ElasticPoolName
```
Comment (thelious1974) [inline PSCoreScripts/Copy-SitefinityDatabase.ps1]: this fixes a PSScriptAnalyzer error

File: PSCoreScripts/Copy-SitefinityDatabase.ps1 (hunk 1 above)
Comment (rivoric) [reply PSCoreScripts/Copy-SitefinityDatabase.ps1]: The line should say
> if ($ENV:RELEASE_RELEASENAME) { 
ie testing if the environment variable exists before trying to split it

Comment (thelious1974) [reply PSCoreScripts/Copy-SitefinityDatabase.ps1]: if() condition has been updated, and a test for it has been added.

**Pull request merged**